website/docs/d/rde_type_behavior.html.markdown

```diff
@@ -0,0 +1,55 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_rde_type_behavior"
+sidebar_current: "docs-vcd-data-source-rde-type-behavior"
+description: |-
+  Provides the capability of fetching the Behavior overrides from an existing RDE Type in VMware Cloud Director.
+---
+
+# vcd\_rde\_type\_behavior
+
+Provides the capability of fetching the Behavior overrides from an existing RDE Type in VMware Cloud Director.
```

Comment:
The "Behavior overrides" sounds strange. Is there a way to rephrase the sentence? For reference, this one is clear:
https://github.com/vmware/terraform-provider-vcd/pull/1074/files#diff-0269665e1810aefda2c551dbe003dcfa3d3ddc77091875474b3e5cf905a25bd5R11

Reply:
Fixed!